src/style-spec/reference/v8.json

```diff
@@ -3755,26 +3737,25 @@
         ]
       },
       "transition": true,
-      "doc": "Controls the strength of the fog effect. Reducing the fog strength pushes the fog farther toward the horizon, leaving more visible space for haze, if enabled. When haze energy is zero, this will usually be 1.0.",
+      "doc": "Controls the strength of fog within the fog range. Reducing the density pushes the fog farther toward the horizon, leaving more visible space for haze, if enabled. When haze opacity is zero, density will usually be 1.0.",
       "sdk-support": {
         "basic functionality": {
           "js": "2.3.0"
         }
       }
     },
-    "haze-energy": {
-      "type": "number",
+    "haze-color": {
+      "type": "color",
       "property-type": "data-constant",
-      "default": 0.0,
-      "minimum": 0.0,
+      "default": "rgba(114,135,213,0)",
       "expression": {
         "interpolated": true,
         "parameters": [
           "zoom"
         ]
       },
       "transition": true,
-      "doc": "Controls the strength of haze in the fog layer. Haze is an additive effect which approximates the look of an atmosphere. Increasing this strengthens the haze. Reducing the fog strength pushes fog toward the horizon, making haze visible.",
+      "doc": "The color of the haze component of fog. Alpha adjusts amount of haze.",
```

Comment:
For the discoverability of this feature, do you think that we should mention the default in the documentation as well? I'm just afraid of users not being able to immediately find haze due to the alpha default of rgba(114,135,213,0).

Reply:
Like "Haze is disabled (zero opacity) by default."

Reply:
Or even more direct:
>Alpha adjusts the amount of haze, note that haze is disabled by default (zero alpha) make sure to increase that value to enable this effect.